the-leaf-handbook.md

```diff
@@ -74,6 +74,18 @@ Where do you think you could develop yourself professionally to improve your ski
 
 Lastly, you may also want to consider some ideas for goals, although these will be worked on properly together with your manager during the meeting.
 
+### One on ones (1:1s)
+1:1s are a 30 minute informal call every 2 weeks between you and your manager. They provide an opportunity to find out how you are, and give even the smallest grievance a chance to be heard. Use your 1:1 as an opportunity to:
+
+- Discuss how you're feeling.
+- Discuss how your work has been going.
+- Surface any specific problems or challenges.
+- Highlight specific triumphs or learnings.
```

Comment:
"Specific" used in the bullet before; I'd suggest using "any" _(Highlight any triumphs or learnings.)_